omero/sysadmins/limitations.txt

```diff
@@ -26,6 +26,29 @@ correctly. OpenJDK and OracleJDK versions 6 and 7 are also supported.
 Windows OS issues
 -----------------
 
+Failure response on import for new databases
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you have import failures with an error such as::
+
+   java.lang.RuntimeException: Failure response on import!
+   Category: ::omero::grid::ImportRequest
+   Name: import-request-failure
+   Parameters: {stacktrace=java.lang.RuntimeException: omero.ValidationException
+       serverStackTrace = "ome.conditions.ValidationException: could not insert: [ome.model.core.Pixels];
+
+when trying to import data to a new database (or possibly one just upgraded to
+5.1), it may be an issue with the database initialization. If you are working
```

Comment:
Maybe be a bit more specific here about what the "issue" is.  If it's e.g. due to using the wrong encoding for the database it should state that rather than leaving the reader guessing!